makefile

```diff
@@ -172,8 +172,11 @@ vision-torch-tf-setup: env
 
 	@$(PIP) install -q "tensorflow-hub==0.12.0";
 
+nlp-tests-setup: env
```

Comment:
Why not in nlp req file?

Reply:
Because I don't want to require packages that are needed only for properties.